packages/components/src/focal-point-picker/index.js

```diff
@@ -210,27 +211,31 @@ export class FocalPointPicker extends Component {
 				</div>
 				<div className="components-focal-point-picker_position-display-container">
 					<BaseControl label={ __( 'Horizontal Pos.' ) }>
-						<input
-							className="components-text-control__input"
-							id={ horizontalPositionId }
-							max={ TEXTCONTROL_MAX }
-							min={ TEXTCONTROL_MIN }
-							onChange={ this.horizontalPositionChanged }
-							type="number"
-							value={ this.fractionToPercentage( percentages.x ) }
-						/>
+
+					<NumberControl
```

Comment:
It looks like here we have a NumberControl inside a BaseControl. NumberControl renders its own BaseControl component so I think the existing BaseControl can also be removed.

Reply:
Thanks for looking @jorgefilipecosta! I'm open to any tips or ideas here. I'm pretty new to the components code. 

Yeah, we could do away with the BaseControl around each input here with the focal-point-picker. Will have to restyle though to handle the `<span>%</span>` appended to each.

Might even be worth adding a `numericUnit` prop in the NumberControl to handle it.